src/vm-repair/azext_vm_repair/_params.py

```diff
@@ -48,3 +48,6 @@ def load_arguments(self, _):
 
     with self.argument_context('vm repair list-scripts') as c:
         c.argument('preview', help="URL of forked repair script library's map.json https://github.com/{user}/repair-script-library/blob/master/map.json")
+
+    with self.argument_context('vm repair reset-nic') as c:
+        c.argument('yes', help='Auto starts the VM skipping prompt if it is not running.')
```

Comment:
Could we use a parameter name with clearer meaning, such as `--skip-prompt`?

Reply:
Actually, '--yes' used by multiple az commands as a skip prompt parameter. 

For example: 
https://docs.microsoft.com/en-us/cli/azure/vm?view=azure-cli-latest#az-vm-delete
az vm delete [--force-deletion]
             [--ids]
             [--name]
             [--no-wait]
             [--resource-group]
             [--yes]


AND 
https://docs.microsoft.com/en-us/cli/azure/deploymentmanager/rollout?view=azure-cli-latest
az deploymentmanager rollout restart --name
                                     --resource-group
                                     [--skip-succeeded]
                                     [--yes]

and many more commands. I was following consistency among commands. I think --yes is reasonable.

Reply:
But I edited the param description to be more readable.